ydb/docs/en/core/concepts/datamodel/_includes/table.md

```diff
@@ -19,6 +19,8 @@ Please note that currently, the `NOT NULL` constraint can only be applied to col
 
 {{ ydb-short-name }} supports the creation of both row-oriented and column-oriented tables. The primary difference between them lies in their use-cases and how data is stored on the disk drive. In row-oriented tables, data is stored sequentially in the form of rows, while in column-oriented tables, data is stored in the form of columns. Each table type has its own specific purpose.
 
+{% include [#{T}](./table-name-rules.md) %}
```

Comment:
Didn't you plan to include this somewhere else too, like in `create table` docs? Otherwise, I'd make it inline and maybe just give a link.